pkg/server/serverpb/status.proto

```diff
@@ -83,8 +83,12 @@ message DetailsRequest {
   // node_id is a string so that "local" can be used to specify that no
   // forwarding is necessary.
   string node_id = 1;
-  // ready specifies whether the client wants to know whether a node is ready to
-  // receive traffic. If a node is unready, an error will be returned.
+  // ready specifies whether the client wants to know whether the
+  // target node is ready to receive traffic. If a node is unready, an
+  // error will be returned.
+  //
+  // Note: this is a more general version of the ready flag on
+  // HealthRequest.
```

Comment:
Can't we just remove this? Users were always hitting `/health` directly (or at least I don't think we ever documented anything but), and those requests will go elsewhere now. If we're comfortable removing this I think we would want to.